app/services/foreman/global_id.rb

```diff
@@ -0,0 +1,25 @@
+module Foreman
+  module GlobalId
+    ID_SEPARATOR = '-'
+    VERSION_SEPARATOR = ':'
+    DEFAULT_VERSION = 1
```

Comment:
Do we plan to do API versioning? From what I [read](https://graphql.org/learn/best-practices/#versioning), GraphQL rather goes the 'evolution' way of deprecating old fields and adding new ones...

Reply:
Let's figure it out together. Right now I'd consider the graphql api highly experimental. This versioning was just added to be future proof.

Reply:
Seems reasonable.